src/api/admin.ts

```diff
@@ -26,7 +26,7 @@ app.post('/sslCerts', async (req, res) => {
       return acc + `${key}=${value} `
     }, '')
     const { stderr } = await exec(
-      `${envVars} ./acme.sh/acme.sh --issue --dns ${service} -d "*.${selectedDomain}" -d ${selectedDomain} --force`
+      `${envVars} ./acme.sh/acme.sh --issue --dns ${service} -d ${selectedDomain} -d "*.${selectedDomain}" --force`
```

Comment:
when you type this into the command line, it only creates one cert. Here's what I think we need to do:
```
const { stderr } = await exec(
  `${envVars} ./acme.sh/acme.sh --issue --dns ${service} -d ${selectedDomain} --force`
)
... await exec(
  `${envVars} ./acme.sh/acme.sh --issue --dns ${service} -d *.${selectedDomain} --force`
)
```